agent/build.gradle

```diff
@@ -3,6 +3,11 @@ import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
 plugins {
     id("com.github.johnrengelman.shadow") version "6.0.0"
 }
+
+apply from: "$rootDir/gradle/shadow.gradle"
+
+def relocatePackages = ext.relocatePackages
```

Comment:
Is this local variable needed? Or can you just directly use `ext.relocatePackages` inside shadowJar task?

Reply:
I think it is, gradle just blows up when I use `ext.relocatePackages` directly:

```
> No signature of method: instrumentation_7xo1njze01y8apwpbo8ndab43.shadowJar() is applicable for argument types: (instrumentation_7xo1njze01y8apwpbo8ndab43$_run_closure3) values: [instrumentation_7xo1njze01y8apwpbo8ndab43$_run_closure3@242f6e99]
```